src/domain/safe-apps/entities/__tests__/safe-app.factory.ts

```diff
@@ -0,0 +1,30 @@
+import { faker } from '@faker-js/faker';
+import { SafeAppAccessControl } from '../safe-app-access-control.entity';
+import { SafeAppProvider } from '../safe-app-provider.entity';
+import { SafeApp } from '../safe-app.entity';
+import safeAppAccessControlFactory from './safe-app-access-control.factory';
+import safeAppProviderFactory from './safe-app-provider.factory';
+
+export default function (
+  id?: number,
+  url?: string,
+  name?: string,
+  iconUrl?: string,
+  description?: string,
+  chainIds?: number[],
+  provider?: SafeAppProvider,
+  accessControl?: SafeAppAccessControl,
+  tags?: string[],
+): SafeApp {
+  return <SafeApp>{
+    id: id ?? faker.datatype.number(),
+    url: url ?? faker.internet.url(),
+    name: name ?? faker.random.word(),
+    iconUrl: iconUrl ?? faker.internet.url(),
+    description: description ?? faker.random.word(),
+    chainIds: chainIds ?? [faker.random.numeric(), faker.random.numeric()],
+    provider: provider ?? safeAppProviderFactory(),
+    accessControl: accessControl ?? safeAppAccessControlFactory(),
+    tags: tags ?? false,
```

Comment:
Should be an array of string.

Reply:
Yes, good catch!

Reply:
Changed in [21ff50f](https://github.com/5afe/safe-client-gateway-nest/commit/21ff50f7bc77e3cb11bcbaa2c1cf0322c60cd08c)